polymer/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>abe-polymer2</title>
+    <meta name="description" content="test of Polymer">
+
+    <!-- See https://goo.gl/OOhYW5 -->
+    <link rel="manifest" href="/manifest.json">
```

Comment:
I think this file does not exist yet?

Reply:
Yes, it's generated by AMPByExample, it's inside the main dist directory.